posts/2021-01-29-airports-at-scale.adoc

```diff
@@ -0,0 +1,39 @@
+= Airports at Scale
+nMoncho
+v1.0, 2020-01-29
+:title: Airports at Scale
+:tags: [database, nosql, big data, modelling, programming]
+
+== Assessing NoSQL Databases
+
+In a https://blog.lunatech.com/2018/11/lunatech-airport-assessment/[previous blog post], I explained the assessment we were using at Lunatech, detailing a solution using a common tech stack, Java with Spring Boot and PostgreSQL. An idea mentioned was that it would’ve been possible to also develop a solution using a NoSQL database, since we did receive such solutions, this got me thinking: could every NoSQL family easily solve each requirement? What would be the advantages of using them over SQL? And, more importantly, do they fit in the problem and domain?
+
+In this series of posts I want to explore these questions, where each post focuses on each family.
```

Comment:
I'm not sure family is fitting here as it does read a bit odd to me. Maybe section? topic?

Reply:
Reading down below I get the family verbiage now. It might be better to be explicit here and say
```suggestion
In this series of posts I want to explore these questions, where each post focuses on each NoSQL family.
```

Reply:
> In this series of posts I want to explore these questions, where each post focuses on each NoSQL family.

You might be more specific I think, like:

In this series of posts I want to explore these questions, where each post focuses on a specific member of the  NoSQL family.

Reply:
> I'm not sure family is fitting here as it does read a bit odd to me. Maybe section? topic?

I switched to *category*, probably this is coming from Spanish background